packages/xod-client-electron/src/arduinoDependencies/messages.js

```diff
@@ -38,4 +38,10 @@ export default {
     solution: 'Now you are able to upload the program',
     persistent: true,
   }),
+  ARDUINO_CLI_NOT_FOUND: ({ path }) => ({
+    title: 'Arduino-cli not found',
+    note: `Tried to find arduino-cli by the path: ${path}`,
+    solution:
+      'If you run XOD IDE in the DEV mode you have to set the path to the arduino-cli in your $PATH or in the env variable "ARDUINO_CLI"',
```

Comment:
We can pass `isDev` as an argument and adjust the phrase accordingly.

```
isDev
  ? 'When running in the development mode `arduino-cli` should be available on $PATH or explicitly set with $XOD_ARDUINO_CLI environment variable'
  : 'This is a bug, report it to XOD developers'
```

Yes, I overlooked it in the original comment, but we should scope our environment variables with `XOD_` prefix.